src/theory/quantifiers/cegqi/ceg_arith_instantiator.cpp

```diff
@@ -267,7 +266,7 @@ bool ArithInstantiator::processAssertion(CegInstantiator* ci,
           << pvmod << " -> " << uval << ", styp = " << uires << std::endl;
     }
     // take into account delta
-    if (ci->useVtsDelta() && (uires == 2 || uires == -2))
+    if (uires == 2 || uires == -2)
```

Comment:
Can `uires` (and `ires`) take on any value or only a specific set of values? If the latter, it may be a good idea to introduce macros for the values and give them names that hint at what the values mean.

Reply:
I agree, looking into this a bit, this will be a fairly major refactor, I'll do this on a follow up PR.

Reply:
SG